tools/make_nr/make_nr.py

```diff
@@ -141,7 +141,7 @@ def make_nr(input_fasta, output_fasta, sep=";", sort_ids=False):
         )
     elif len(input_fasta) == 1:
         # Single file, no need to make a copy or edit titles
-        os.symlink(os.path.abspath(input_fasta), output_fasta)
```

Comment:
Doh. I clearly wasn't thinking there. I was right to want to add a test though 🤣